manifests/init.pp

```diff
@@ -80,7 +80,8 @@
   $key             = undef,
   $ipv6            = undef,
   $bind_ip         = undef,
-  $pidfilepath     = undef
+  $pidfilepath     = undef,
+  $pidfilemode     = undef
```

Comment:
Shouldn't this be passed to mongodb::server in the body of the class as well? Otherwise there is no value in exposing the parameter in init.